### ISSUE TITLE
Template: Keyboard issues with Hamburger Menu

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#87]
In mobile view, the Hamburger menu allows content behind it to receive focus AND the "X" button to exit the content does not receive focus. 

This should Function similar to a modal as it needs to hold focus until a user exits the content.

**Comment by syarwick23** [comment]
Issue fixed with new template
